src/js/tangram-play.js

```diff
@@ -419,13 +419,13 @@ var ReactDOM = require('react-dom');
 
 import MenuBar from './components/menu-bar.react';
 import MapPanel from './components/map-panel.react';
-import FileDrop from './file/drop';
+import OverlaysContainer from './ui/OverlaysContainer';
 
 let mountNode1 = document.getElementById('menu-bar');
 ReactDOM.render(<MenuBar />, mountNode1);
 
 let mountNode2 = document.getElementById('map-panel');
 ReactDOM.render(<MapPanel />, mountNode2);
 
-let mountNode3 = document.getElementById('filedrop');
-ReactDOM.render(<FileDrop />, mountNode3);
+let mountNode3 = document.getElementById('overlays-container');
```

Comment:
We should probably start naming these better? instead of mountNode2, mountNode3...